docs/quickstart/intro.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Redoc quickstart guide
+---
+
+# Redoc quickstart guide
+
+This guide includes step-by-step instructions for how to get started using
+Redoc to render your OpenAPI definition.
+
+Redoc offers multiple options for rendering your OpenAPI definition.
+You should select the option that best fits your needs.
+
+The different options include using the following:
+
+- **[Live demo](/redoc/quickstart/live-demo.md):**
+  The live demo offers a fast way to see how your OpenAPI will render with Redoc.
+- **[HTML element](/redoc/quickstart/html.md):**
+  Using an HTML element works well for typical website deployments.
+- **[React component](/redoc/quickstart/react.md):**
+  Using a React component is an option for users with a React-based application.
+- **[Docker image](/redoc/quickstart/docker.md):**
+  Using a Docker image works in a container-based deployment.
+
```

Comment:
There is also a CLI option where you can build a bundle of HTML/CSS/JS files.

Reply:
ah, I didn't know that - I will add it.